scenario/node/src/customworld.ts

```diff
@@ -95,31 +95,44 @@ function assertDefined<T>(value: T | null | undefined, property: string): T {
     return value;
 }
 
+interface GatewayContext {
+    gateway?: Gateway;
+    identity?: Identity;
```

Comment:
You always have an identity at the point you're creating the GatewayContext so this doesn't need to be optional. And you can save having the `getIdentity()` function to assert it exists.